web/src/main/java/nl/tehdreamteam/se42/web/soap/user/SoapUserController.java

```diff
@@ -11,8 +13,12 @@
 @WebService
 public class SoapUserController implements UserController {
 
+    private static final Logger logger = LogManager.getLogger(SoapUserController.class.getSimpleName());
+
     @Override
     public String login(@WebParam(name = "username") String username, @WebParam(name = "password") String password) {
+        logger.debug("Soap request for login (username='{}', password='{}').", username, password);
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Remove this hard-coded password. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS2068)